beta/src/sidebarReference.json

```diff
@@ -230,6 +230,66 @@
               "wip": true
             }
           ]
+        },
+        {
+          "title": "ReactDOM Attributes",
+          "path": "/apis/dom-attributes/domattributes",
+          "routes": [
+            {
+              "title": "aria",
+              "path": "/apis/dom-attributes/aria"
+            },
+            {
+              "title": "checked",
+              "path": "/apis/dom-attributes/checked"
```

Comment:
Same here, what if all of these were like `/apis/react-dom/attributes/checked`

Reply:
This makes sense. I will change it. 

Just one query-
So, these APIs will also get added below ReactDom API in the UI is that ok?